### DISCUSSION TITLE
[Feature] Upload button on mobile side

### DISCUSSION THREAD

**Discussion by abbbubble** [immich-app/immich#14720] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When multiple devices are logged into the same account, some mobile devices may only need to upload a few photos, but there’s no separate upload button.

So, I suggest:
Adding a separate upload button in the app for easier use.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7075
